sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.0-beta.2 (2023-06-30)
 
 ### Other Changes
 
+- Bump ResourceManager version to 2021-12-01-preview
+- Update variable naming
```

Comment:
This can be removed since we did not rename the existing types, just the newly added types in the new version.